.github/workflows/ci.yml

```diff
@@ -8,11 +8,11 @@ concurrency:
 
 jobs:
   tests:
-    continue-on-error: true
```

Comment:
Disable the `continue-on-error` setting. This caused the workflow to always pass, even if a job failed.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error